lib/index.js

```diff
@@ -459,7 +460,8 @@ module.exports = class Coins {
           , responseField = options.responseField || _url
           , version = options.version || "d/api"
           , qs = querystring.stringify(params)
-          , url = this.host + version + "/" + _url + "/" + (qs ? "?" + qs : "")
+          , endWithSlash = options.endWithSlash || true
```

Comment:
I'm was wrong with this condition because this would always be true.

It should be: `, endWithSlash = options.endWithSlash == null ? true : options.endWithSlash`

Reply:
@iceniveth I indicate to remove trailing slash option instead. By default, every url should have a trailing slash at the end of the url.